lib/csf-tools/src/CsfFile.ts

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable no-underscore-dangle */
+import fs from 'fs-extra';
+import { parse } from '@babel/parser';
+import generate from '@babel/generator';
+import * as t from '@babel/types';
+import traverse, { Node } from '@babel/traverse';
+import { toId, isExportStory } from '@storybook/csf';
+
+const logger = console;
+
+interface Meta {
+  title?: string;
+  includeStories?: string[] | RegExp;
+  excludeStories?: string[] | RegExp;
+}
+
+interface Story {
+  id: string;
+  name: string;
+  parameters: Record<string, any>;
+}
+
+const getMeta = (declaration: any): Meta => {
+  // { title: 'asdf', includeStories: /.../ (or []), excludeStories: ... }
+  const meta: Meta = {};
+  declaration.properties.forEach((p: Node) => {
+    if (t.isObjectProperty(p) && t.isIdentifier(p.key)) {
+      if (p.key.name === 'title') {
+        meta.title = parseTitle(p.value);
+      } else if (['includeStories', 'excludeStories'].includes(p.key.name)) {
+        // @ts-ignore
+        meta[p.key.name] = parseIncludeExclude(p.value);
+      }
+    }
+  });
+  return meta;
+};
+
+const parseTitle = (value: any) => {
+  if (value.type === 'StringLiteral') {
+    return value.value;
+  }
+  return undefined;
+};
+
+function parseIncludeExclude(prop: any) {
+  const { code } = generate(prop, {});
+  // eslint-disable-next-line no-eval
+  return eval(code);
```

Comment:
Can we avoid the `eval` here with `new Regexp()` or something? Are we concerned about `eval`s?